doc/manual/utils.nix

```diff
@@ -56,6 +56,21 @@ rec {
       body = ''
           ${description}
 
+        '' + (if experimental-feature != null then ''
```

Comment:
I feel like string interpolation looks cleaner, but feel free to ignore.
In Nixpkgs I would also say `lib.optionalString`, but that may not apply here.

Reply:
I agree with that, and recommend following my example of creating helper functions with self-explanatory names when inline expressions get out of hand.

Reply:
Added `optionalString` in a preparatory commit, and used it in a bunch of other places too.